test/libsolidity/smtCheckerTests/operators/div_zero.sol

```diff
@@ -5,4 +5,4 @@ contract C {
 	uint x = 2 / z;
 }
 // ----
-// Warning 6084: (69-74): BMC: Division by zero happens here.
```

Comment:
It seems there were 2 different error codes for BMC Division by zero errors: 6084 and 3046. What was the difference?

Reply:
Looks like 6084 is used whenever there's no callstack, not only for div by 0.